src/NHibernate.Test/SystemTransactions/DistributedTransactionFixture.cs

```diff
@@ -485,6 +436,74 @@ public void NH1744()
 		}
 
 		[Test]
+		public void CanUseSessionWithManyScopes([Values(false, true)] bool explicitFlush)
+		{
+			// Note that this fails with ConnectionReleaseMode.OnClose a SqlServer:
+			// System.Data.SqlClient.SqlException : Microsoft Distributed Transaction Coordinator (MS DTC) has stopped this transaction.
+			using (var s = OpenSession())
+			//using (var s = Sfi.WithOptions().ConnectionReleaseMode(ConnectionReleaseMode.OnClose).OpenSession())
```

Comment:
Yuck, one more trouble. I do not see anyway to fix that, excepted not using the old `OnClose` connection release mode. Since documentation advises against using it, I think we are fine not fixing that trouble.